src/pages/index.js

```diff
@@ -4,91 +4,195 @@ import "../pages/index.css";
 import UserInfo from "../components/UserInfo.js";
 import Section from "../components/Section.js";
 import ImagePopup from "../components/PopupWithImage.js";
+import PopupConfirm from "../components/PopupConfirm.js";
```

Comment:
Needs correction: I can't update the avatar

Reply:
I can update the avatar on my end.  Please see file attached.

https://github.com/jmore2290/se_project_aroundtheus/assets/4120656/0dc7e8b3-b7fb-4604-8433-f948c8a2e5f8